content/influxdb/v2.0/influxdb-templates/cloud.md

```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
 products: [cloud]
 ---
 
-Use the [`influx` command line interface (CLI)](/influxdb/v2.0/reference/cli/influx/)
-to apply and manage templates in your InfluxDB Cloud account.
+To use templates in InfluxDB Cloud, you have a couple options:
+
+- [Install and customize templates directly in the Cloud UI](#install-and-customize-a-template-in-the-cloud-ui)
+- [Apply and manage templates using the `influx` CLI](#apply-and-manage-templates-using-the-influx-cli)
+
+## Install and customize a template in the Cloud UI
+
+{{% note %}}
+To ensure the template works, complete the **ReadMe** instructions after installing a template. For example, you may need to create a bucket with a specific name in your Cloud instance before the template can be used.
+{{% /note %}}
+
+1. Under **Settings** icon in the navigation bar.
+
+    {{< nav-icon "settings" >}}
 
+2. Click **Templates**.
+3. (Optional) If you already have a template URL, you can skip this step. Click **Browse Community Templates** to explore templates and obtain a template URL (YAML or JSON file).
+4. Enter the URL to the template's YAML or JSON file (from the [InfluxData Community Templates](https://github.com/influxdata/community-templates) repository), and then click **Lookup Template**.
+The Template Installer displays the template name and the number of resources included in the template (for example, dashboards, Telegraf configurations, buckets, and more).
+5. Click **Install Template**. A message confirms the template has been successfully installed.
+6. Under **Installed Templates**, find the template, and do the following:
+   - Click **View Readme**. Complete the instructions to finalize your template.
+   - (Optional) To customize a template, under **Installed Resources**, expand the template resources, and then click a resource to update it. Update any of the following resources included in the template:
+
+    - Buckets. [Update buckets](/influxdb/v2.0/organizations/buckets/update-bucket/)
+    - Dashboards. [Control dashboards](/influxdb/v2.0/visualize-data/dashboards/control-dashboard/)
+    - Labels. [Edit labels](/influxdb/v2.0/visualize-data/labels/#edit-a-label)
+    - Checks. [Update checks](/influxdb/v2.0/monitor-alert/checks/update/)
+    - Notification endpoints. [Update notification endpoints](/influxdb/v2.0/monitor-alert/notification-endpoints/update/)
+    - Notification rules. [Update notification rules](/influxdb/v2.0/monitor-alert/notification-rules/update/)
+    - Telegraf configurations. [Update a Telegraf configurations](/influxdb/v2.0/telegraf-configs/update/)
+
+7. (Optional) Add or delete resources from the template, and then use the `influx` CLI to [export resources to a new template (JSON or YAML file)](/influxdb/v2.0/influxdb-templates/create/#export-all-resources).
```

Comment:
Figured links to add/delete resources docs were extra, so left them out. Will include if we feel it's better to be explicit here.